core/services/flux_monitor.go

```diff
@@ -0,0 +1,342 @@
+package services
+
+import (
+	"chainlink/core/eth"
+	"chainlink/core/logger"
+	"chainlink/core/store"
+	"chainlink/core/store/models"
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+	"github.com/shopspring/decimal"
+	"go.uber.org/multierr"
+)
+
+// FluxMonitor is the interface encapsulating all functionality
+// needed to listen to price deviations and new round requests.
+type FluxMonitor interface {
```

Comment:
The golang convention advocates the `er` if you're wrapping a single method:

```go
type Caller interface {
  func Call()
}
```

It does not speak to adding `er` to a group of methods, none of which have the prefix `FluxMonitor`

Reply:
https://golang.org/doc/effective_go.html#interface-names

> By convention, one-method interfaces are named by the method name plus an -er suffix or similar modification to construct an agent noun: Reader, Writer, Formatter, CloseNotifier etc.

> There are a number of such names and it's productive to honor them and the function names they capture. Read, Write, Close, Flush, String and so on have canonical signatures and meanings. To avoid confusion, don't give your method one of those names unless it has the same signature and meaning. Conversely, if your type implements a method with the same meaning as a method on a well-known type, give it the same name and signature; call your string-converter method String not ToString.